### ISSUE TITLE
Template for resources.js

### ISSUE THREAD

**Issue by parasyte** [melonjs/generator-melonjs#4]
The resources.js file can be templated to automatically populate the JSON structure by reading the data directory tree. See also the related ticket in the boilerplate: https://github.com/melonjs/boilerplate/issues/16

**Comment by ellisonleao** [comment]
Grunt can handle this, but i've got 2 images which doesn't belong to the game (icon files for web manifest) . Should we move those images to another folder?  Another question: Perhaps isn't the time to make the generator game code the same as the boilerplate or vice versa?

**Comment by parasyte** [comment]
Well, I just finished this one for the boilerplate : https://github.com/melonjs/boilerplate/issues/16

I won't have any feedback for you on the idea of merging generator and boilerplate. I haven't used either of these, and I think they both have their own advantages. Anyway if it makes sense, then yes maybe merge the projects.

**Comment by ellisonleao** [comment]
I guess we can have the same game code running on both projects. The only thing that will diff is the way both are installed.  I am saying this because the boilerplate code is diverging from this one and i don't think that is cool. It can confuse  the users.

**Comment by parasyte** [comment]
Yes, I agree it's bad to confuse users. Is there a way to merge them after all? Yeoman uses templates, and the boilerplate generally just uses grunt tasks to filter and augment source files.

**Comment by ellisonleao** [comment]
Yes, the best choice is to have the yeoman template a git submodule from the boilerplate project, but i have to check if that is possible.. then check if the grunt works as expected (i call the `grunt` command after the project is created on local folder)
